esp/src/src-react/components/GetLogicalFilePart.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from "react";
+import { DefaultButton, DetailsList, DetailsListLayoutMode, IColumn } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { csvParse } from "d3-dsv";
+import { DaliService, WsDali } from "@hpcc-js/comms";
+import { scopedLogger } from "@hpcc-js/util";
+import { TableGroup } from "./forms/Groups";
+import nlsHPCC from "src/nlsHPCC";
+import { HolyGrail } from "../layouts/HolyGrail";
+
+class MyDaliService extends DaliService {
+    GetLogicalFilePart(request: WsDali.GetLogicalFilePartRequest): Promise<WsDali.ResultResponse> {
+        return this._connection.send("GetLogicalFilePart", request, "json", false, undefined, "ResultResponse");
+    }
+}
+
+const logger = scopedLogger("src-react/components/GetLogicalFilePart.tsx");
+
+const myDaliService = new MyDaliService({ baseUrl: "" });
+
+interface GetLogicalFilePartProps {
+}
+
+export const GetLogicalFilePart: React.FunctionComponent<GetLogicalFilePartProps> = ({
+
+}) => {
+
+    const [columns, setColumns] = React.useState<IColumn[]>([]);
+    const [items, setItems] = React.useState<any[]>([]);
+    const [fileName, setFileName] = React.useState<string>("");
+    const [partNumber, setPartNumber] = React.useState<number>();
```

Comment:
This should probably have a default value, `...useState<number>(1);`, or 0(?), whatever makes sense.